### ISSUE TITLE
Site Redesign

### ISSUE THREAD

**Issue by webdesserts** [react-boilerplate/site#3]
### Landing Page

![react-boilerplate-site_day3](https://cloud.githubusercontent.com/assets/916253/14378412/61856fd0-fd3a-11e5-863d-191e6a8894b5.png)

Wording for Features could probably use some work. For example "Routing" and "Offline First" could probably be expanded a bit. Also Getting Started will need to be updated to reflect the new CLI
## Documentation

![react-boilerplate-site_docs_day3](https://cloud.githubusercontent.com/assets/916253/14378419/695c5020-fd3a-11e5-8376-b4e659d8a351.png)
### React Guilloche

![react-boilerplate-guilloche](https://cloud.githubusercontent.com/assets/916253/14341295/30536604-fc56-11e5-904b-dd6ef01c0d53.png)
### Fonts

**Headings**: Futura PT (Book)
**Body Copy**: Tisa Sans Pro (Regular)
**Code**: Source Code Pro (Regular)

**Suggested Fallbacks (Untested)**:

**Headings**: Helvetica, Arial, sans-serif
**Body Copy**: Helvetica, Arial, sans-serif
**Code**: Courier New, Courier, monospaced
### Colors

![react-boilerplate-site](https://cloud.githubusercontent.com/assets/916253/14378459/a412a4e4-fd3a-11e5-96d8-57fc26a6ee7b.png)

```
$brown: #715c57;
$blue: #7d93aa;
$light-blue: #dae6f1;
$red: #d77158;
$light-red: #f0af9f;
$white: #fafcff;
```

**Comment by oliverturner** [comment]
This is absolutely awesome! _Great_ work @webdesserts!

**Comment by brotzky** [comment]
Fantastic mockups!

**Comment by webdesserts** [comment]
@brotzky Source Code Pro is open source, Tisa Sans Pro and Futura PT are not. I do, however, have a typekit account that I was going to use for the project, so it shouldn't cost anyone anything.

**Comment by webdesserts** [comment]
Also, thanks @oliverturner & @brotzky for the kind words 😊

**Comment by mxstbr** [comment]
Don't know why I didn't comment here, I **love** the design. Amazing work @webdesserts!

**Comment by vigzmv** [comment]
Fixed and merged with #8
